### PR TITLE
relay latency stat

### PR DESCRIPTION
**Pull request by rf**
@Raynos @jcorbin 

Did I do it right? This is just v0 of this concept; v1 would be a stat for each of: in-out, out-service, out-in, in-client.

### CODE REVIEW

File: node/channel.js [hunk 1]
```diff
@@ -132,6 +132,8 @@ function TChannel(options) {
     self.connectionsClosedStat = self.defineCounter('tchannel.connections.closed');
     // self.connectionsBytesRcvdStat = self.defineCounter('connections.bytes-recvd');
 
+    self.relayLatencyStat = self.defineTiming('tchannel.relay.latency');
```
Comment (Raynos) [inline node/channel.js]: This is the bad pattern. use `emitFastStat`

**Comment by Raynos** [comment]
#LEGIT.

**Comment by Raynos** [comment]
#DOIT

**Comment by jcorbin** [comment]
nice

**Pull request merged**